### ISSUE TITLE
Quests not showing reputation. Instead it shows 0 reputation

### ISSUE THREAD

**Issue by IcemanHHW** [Questie/Questie#4028]
## Bug description

Checking the reputation of a quest when hovering over it, always shows 0 reputation instead of the actual reptuation that it rewards. This issue happens on multiple characters, including newly created ones.

Turning off Elvui and everything related to it didn't change anything about the issue.

## Screenshots

![WoWScrnShot_090322_222335](https://user-images.githubusercontent.com/20383130/188286715-45a1efc7-eb8b-4d84-8a3a-07485a608488.jpg)
![WoWScrnShot_090322_222406](https://user-images.githubusercontent.com/20383130/188286719-417fe846-0e51-40f9-bd4e-82e9b07cd6f8.jpg)
![WoWScrnShot_090322_222852](https://user-images.githubusercontent.com/20383130/188286778-3e09b4e6-1104-489c-8558-53cbcda4cdca.jpg)

## Questie & WoW version

Questie version: v7.0.1
WoW version: TBC

**Comment by KoolKeith** [comment]
I confirm this. It's very important for me, want to ride a tiger :)

**Comment by BreakBB** [comment]
Yeah this is a known issue. We are working on it. Thanks for the report! 👍🏻

**Comment by Logonz** [comment]
Some quests also have WAY to much rep


![Image](https://user-images.githubusercontent.com/8261057/188864336-5d16bd80-e205-48b3-b2f3-01b77368cfde.png)

Should be 1000
